e2e/test/scenarios/joins/joins-reproductions.cy.spec.js

```diff
@@ -1135,3 +1138,105 @@ describe("issue 39448", () => {
     });
   });
 });
+
+// See TODO inside this test when unskipping
+describe.skip("issue 27521", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsNormalUser();
+  });
+
+  it("correctly displays joined question's column names (metabase#27521)", () => {
+    cy.visit("/");
+
+    cy.log("Create Q1");
+    newButton("Question").click();
+    entityPickerModal().within(() => {
+      entityPickerModalTab("Tables").click();
+      cy.findByText("Orders").click();
+    });
```

Comment:
Couldn't we `openOrdersTable()` instead?

Use `mode: notebook` if needed.

Reply:
a34e339623a60bb85c1467ff781b0c77a5ac71e6